examples/bzlmod/tests/BUILD.bazel

```diff
@@ -48,6 +48,24 @@ py_test(
     deps = ["//libs/my_lib"],
 )
 
+py_test(
```

Comment:
You could use `bazel_features` in here and set `target_compatible_with` for the `py_test` targets based on the bazel version in that way.

Reply:
Thanks for the tip